simtools/simtel/simtel_config_writer.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/python3
+
+import logging
+
+
+__all__ = ['SimtelConfigWriter']
+
+
+class SimtelConfigWriter:
+    '''
+    SimtelConfigWriter writes sim_telarray configuration files. \
+    It is designed to be used by model classes (TelescopeModel and ArrayModel) only.
+
+    Methods
+    -------
+    writeTelescopeConfigFile(configFilePath, parameters)
+        Writes the sim_telarray config file for a single telescope.
+    writeArrayConfigFile(configFilePath, layout, telescopeModel, siteParameters)
+        Writes the sim_telarray config file for an array of telescopes.
+    writeSingleMirrorListFile(
+        mirrorNumber,
+        mirrors,
+        singleMirrorListFile,
+        setFocalLengthToZero=False
+    )
+        Writes the sim_telarray mirror list file for a single mirror.
+    '''
+    TAB = ' ' * 3
+    SITE_PARS = [
+        'altitude',
+        'atmospheric_transmission',
+        'ref_lat',
+        'ref_long',
+        'array_coordinates',
+        'atmospheric_profile',
```

Comment:
Does sim_telarray need the magnetic field?

Reply:
It doesn't! That is why magnetic_field is on the list of PARS_NOT_TO_WRITE.

Reply:
Ah!